src/main/webapp/i18n/de/studentAnalyticsDashboard.json

```diff
@@ -9,7 +9,8 @@
                 "lectures": "Vorlesungen",
                 "progress": "Fortschritt: {{ progress }}%",
                 "lectureProgress": "Vorlesungsfortschritt: {{ progress }}%",
-                "exerciseProgress": "Aufgabenfortschritt: {{ progress }}%"
+                "exerciseProgress": "Aufgabenfortschritt: {{ progress }}%",
+                "congratulations": "Herzlichen Glückwunsch! Du hast alle freigegebenen Aktivitäten in dieser Kompetenz zu mindestens 80% abgeschlossen 🎉"
```

Comment:
This sounds like that it would be ok to only finish 80% of the lecture units, e.g. only 4 out of 5

Reply:
This is intended because with 80% you get a green score. Students can still finish those, but we won't display it in the dashboard as "next activities", because the learning gain is probably small (Pareto principle)